automatic/git.portable/git.portable.nuspec

```diff
@@ -6,7 +6,7 @@
     <version>2.11.0</version>
     <authors>The Git Development Community</authors>
     <owners>Chocolatey</owners>
```

Comment:
You changed this to lowercase in the other packages should therefore be lower case here too